exporter/awsxrayexporter/config.go

```diff
@@ -30,6 +30,9 @@ type Config struct {
 	// AWS client.
 	MiddlewareID *component.ID `mapstructure:"middleware,omitempty"`
 
+	// X-Ray Export sends spans in its original otlp format to X-Ray Service when this flag is on
+	TransitSpanInOtlpFormat bool `mapstructure:"transit_spans_in_otlp_format,omitempty"`
```

Comment:
I am not sure this config name is accurate because strictly speaking we are not using otlp format. Server side is accepting base64 encoded protobuf binary where OTLP is passing the raw bytes are request body. I suggest we call it something like `encode_to_otel_binary_base64`

Reply:
I agree with the intention but I don't think exposing too much implementation details to users is helpful. All users need to know is that this flag enable them to send complete otlp format span to X-Ray, which contains more information than X-Ray format. Hence, i would still prefer keeping the name.